docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
```

Comment:
Suggestion:

Before contributing to this repository, always submit and discuss your proposed changes through an issue. Should you need further clarification, you can reach out to us on Telegram, or Discord. Please wait to be assigned to an issue before opening a pull request.